### ISSUE TITLE
[NPC][Spell] Paralytic poison

### ISSUE THREAD

**Issue by neheri** [SunwellWoW/Sunwell-TBC-Bugtracker#1607]
Decription: Poison should deal damage every 2 seconds and stun for 2 seconds every 5 seconds.

How it works: It deals damage but doesn't stun.

How it should work: Should deal damage and stun.

Source (you should point out proofs of your report, please give us some source): 
![wowscrnshot_111818_104920](https://user-images.githubusercontent.com/37372869/48670868-9f10ec00-eb1f-11e8-9222-a00a6d1a5d0c.jpg)

**Comment by Mikov1605** [comment]
You're wrong, the spell SHOULD deal damage every 2 sec + stun after 15 seconds, stun duration is 3 sec

**Comment by neheri** [comment]
Yes, I've read to tooltip wrong but so did you. I said it deals damage and doesn't stun and that's how it works at the moment. It should deal damage as it does now but also stun,

It should stun for 5 seconds after 30 seconds.

**Comment by Mikov1605** [comment]
Yeah, i Remember it from woltk, where stun was for 3, not 5. My bad

**Comment by notsofamous** [comment]
Duplicate: https://github.com/SunwellWoW/Sunwell-TBC-Bugtracker/issues/288
